apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -276,6 +276,52 @@ module.exports = {
             },
           ],
         },
+        {
+          label: "Imprint",
+          name: "imprint",
+          widget: "object",
```

Comment:
```suggestion
```
Don't think `widget` is supported at file [level](https://www.netlifycms.org/docs/collection-types/#file-collections).